meeting-minutes/client-minutes/nov10-client-minutes.md

```diff
@@ -0,0 +1,83 @@
+
+- It was mentioned that the form could be split into subsections (one section for obtaining user input with a 'next' button to navigate to the second subsection for account creation). Is this a must for the end-product, or can we have all information on one page?
+  - All of the information can be on one page, so **we don't have to separate the form into subsections**.
+- Can we confirm exactly what fields should be included on Mentee forms, as well as Mentor forms?
+  - **Mentee Forms**
+    - Name, email, list of ranked skills, text-area to obtain a description from the user about themselves/their goals for the program.
+    - The 'Timeframe Seeking Mentorship' field can be removed (later on in development, the client might want to have the feature, but she said it's not needed as of now).
+    - Reminder: Mentees do not need to create an account, and so they do not need a password.
+  - **Mentor Forms**
+    - Name, email, list of ranked skills, text-area to obtain a description from the user about themselves/their goals for the program, some way of obtaining the user's availability, and password for account creation.
+  - NOTE: After discussing what input fields are necessary on each form, the client was contemplating whether we should be collecting the user's time-zone, or if collecting the user's location would suffice. She mentioned that the time-zone could possibly be extracted from the user's browser, but location alone might suffice.
+- A road block from this week was the complexity that matching based on common availability introduces. Is common availability **required** when it comes to making mentee and mentor matches, or can we match primarily based on skills and time-zones?
+  - The client recognizes the complexity that matching based on common availability might have. As such, when creating mentee and mentor matches, **common availability does not need to be considered**.
+    - The client's recommended course of action: The process can be analogous to a job recruiting process (i.e The individual conducting interviews sends out a calendar to the interviewee. The interviewee receives a calendar of available time slots, and picks one). Therefore, for our project we can:
+      - Obtain availability for Mentors only.
+      - Once a match has been made, send the Mentor's availability to the Mentee.
+      - The mentee can then select a timeslot based on when the Mentor is available.
+
+The client asked how our team was progressing with our use of Firebase, so the MentorshipForm.js file was shown since authentication was worked on over the last sprint. The client recommended also initializing Firebase within App.js as well (so that all the components being called from App.js have access to the information in the database).
```

Comment:
I am not sure about this. As things stand, firebase is initialized in its own file, firebase-init.js, and any component which needs info from firebase can simply ```import firebase from '../firebase-init';```and use firebase.anyFunction. Is that acceptable?

Reply:
I did show the firebase-init.js file to the client, but I wasn't completely aware of the logistics surrounding the file. If that's the case, I think our current setup should be alright then.